__init__.py

```diff
@@ -805,6 +817,7 @@ def playlists(self):
 
     def refresh_saved_tracks(self):
         """Saved tracks are cached for 4 hours."""
+        return []
```

Comment:
unimplemented?

Reply:
This looks like a mistake on my part not sure I intended this to be included...will fix

Reply:
this was fixed :heavy_check_mark: